core/api/jrpc/value_converter.hpp

```diff
@@ -207,7 +207,12 @@ namespace kagome::api {
   }
 
   inline jsonrpc::Value makeValue(const primitives::Justification &val) {
-    return makeValue(val.data);
+    jArray frnk;
+    frnk.emplace_back(jArray({'F', 'R', 'N', 'K'}));
+    frnk.emplace_back(jArray(val.data.begin(), val.data.end()));
+    jArray res;
+    res.emplace_back(std::move(frnk));
+    return res;
```

Comment:
What should the result look like?
`[ ['F', 'R', 'N', 'K'], ['data'] ]` or `['F', 'R', 'N', 'K', 'data']`? If the second is the correct option, then it seems we could return `frnk`. Otherwise, all is ok.